.github/workflows/main.yml

```diff
@@ -0,0 +1,20 @@
+name: trigger engine jenkins job
+on:
+  # Trigger the workflow on push or pull request,
```

Comment:
Why does this say `push or pull request`?

Reply:
I amended the commit